diff --git a/meson.build b/meson.build
index 2a26fbb..817c772 100644
--- a/meson.build
+++ b/meson.build
@@ -11,9 +11,8 @@ project(
     license: 'BSD-2-Clause',
 )
 
-dlibdir = get_option('libdir') / 'dinit'
-tmpfdir = get_option('libdir') / 'tmpfiles.d'
-srvdir = get_option('libdir') / 'dinit.d'
+dlibdir = '/usr/lib/dinit'
+srvdir = '/usr/lib/dinit.d'
 earlydir = srvdir / 'early'
 
 kmod_dep = dependency('libkmod')
@@ -157,20 +156,3 @@ install_data(
     install_dir: dlibdir,
     install_mode: 'rwxr-xr-x',
 )
-
-tmpfiles = [
-    'chimera.conf',
-    'home.conf',
-    'static-nodes-permissions.conf',
-    'tmp.conf',
-    'var.conf',
-    'x11.conf',
-]
-
-foreach tmpf: tmpfiles
-    install_data(
-        'tmpfiles/' + tmpf,
-        install_dir: tmpfdir,
-        install_mode: 'rw-r--r--',
-    )
-endforeach
